addon/components/paper-button.js

```diff
@@ -7,14 +7,22 @@ import ColorMixin from 'ember-paper/mixins/color-mixin';
 const { computed } = Ember;
 
 export default BaseFocusable.extend(RippleMixin, ProxiableMixin, ColorMixin, {
-  attributeBindings: ['type'],
   type: 'button',
-  tagName: 'button',
+  tagName: computed('href', function() {
```

Comment:
We're on the same page. Technically you should be able to use a CP without a dependent key. I'll try that, but I bet it still raises a deprecation. I just hate that init function, but it's an aesthetic thing, not a functional or technical objection.